src/control.c

```diff
@@ -19,11 +19,19 @@ static volatile float kp_linear = 1600.;
 static volatile float kd_linear = 100.;
 static volatile float kp_angular = 30.;
 static volatile float kd_angular = 50.;
+static volatile float ki_angular_front = 50.;
+static volatile float ki_angular_side = 40.;
+static volatile float side_sensors_error_factor = 50.;
+static volatile float front_sensors_error_factor = 20.;
 
 static volatile int32_t pwm_left;
 static volatile int32_t pwm_right;
 
 static volatile bool collision_detected_signal;
+static volatile bool side_sensors_control_enable;
+static volatile bool front_sensors_control_enable;
```

Comment:
I would say "enabled" rather than "enable".

Reply:
Fixed, they are not volatile either.

Reply:
I think they are. :blush: